README.md

```diff
@@ -182,11 +182,17 @@ Some of the Trillian Go code is autogenerated from other files:
 Re-generating mock or protobuffer files is only needed if you're changing
 the original files; if you do, you'll need to install the prerequisites:
 
-  - `mockgen` tool from https://github.com/golang/mock
-  - `stringer` tool from https://golang.org/x/tools/cmd/stringer
-  - `protoc`, [Go support for protoc](https://github.com/golang/protobuf),
-     [grpc-gateway](https://github.com/grpc-ecosystem/grpc-gateway) and
-     [protoc-gen-doc](https://github.com/pseudomuto/protoc-gen-doc).
+  - a series of tools, using `go install` to ensure that the versions are
+    compatible and tested:
+
+    ```
```

Comment:
Should we make it clear that these commands need to be executed from the `trillian` dir?
```
cd $(go list -f '{{ .Dir }}' github.com/google/trillian)
```

Reply:
Excellent suggestion. Done!